### DISCUSSION TITLE
Error ElementResizeObserver ...

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#5614] [Q&A] [answered]
Hi all I have a bad error string in my application which however does not affect the functioning of my app,
"undefine \n at ElementResizeObserver etc. etc."
I'd like to understand where it comes from, and if it has already happened to anyone.
I attach the image of the error

I don't know if I'm posting the problem in the correct channel, or should I move it under "itwin-Ui"

Thanks for the precious help.
<img width="936" alt="error" src="https://github.com/iTwin/itwinjs-core/assets/57446652/ed680a71-c239-44b3-a191-33494190cf32">

**Comment by grigasp** [comment, marked as answer]
Hi, can you please specify which version of `@itwin/property-grid-react` you're using? The call stack suggests is an old one - could you try updating to at least `0.6`?

**Comment by Paolo-lab** [comment]
hello I'm using version 0.5.5, now I try to update it

Thank you
